src/validation/permission-exclusions.ts

```diff
@@ -76,11 +78,12 @@ export class PermissionExclusionsValidator {
     if (permissionExclusions.length <= 0) {
       return permissionExclusionsSet
     }
+
     if (isPermissionExclusionCombos(permissionExclusions[0])) {
       const castedPermissionExclusions = <PermissionExclusionCombos[]>permissionExclusions
       castedPermissionExclusions
         .filter(item => item.permissionA === permission)
-        .forEach(perm => permissionExclusionsSet.add(perm.permissionB))
+        .forEach(perm => permissionExclusionsSet.add(perm.permissionB || ''))
```

Comment:
Why is empty string there?

Reply:
because `permissionB` might be _underfined_:
https://github.com/mojaloop/security-role-perm-operator-svc/pull/23/files#diff-a474623775855ac8b1508f372d20c27a8ad04ef1a5edd8f5df361ae7f5c740b0R219

Reply:
Ok